samples/subsys/power/power_mgr/CMakeLists.txt

```diff
@@ -4,3 +4,4 @@ project(power_mgr)
 
 FILE(GLOB app_sources src/*.c)
 target_sources(app PRIVATE ${app_sources})
+target_include_directories(app PRIVATE $ENV{ZEPHYR_BASE}/subsys/power)
```

Comment:
what exactly do you need from here? We should not include non-public APIs in samples.

Reply:
I have removed these, now that Rama's #11656 is merged